QuizApp/Login/LoginView.swift

```diff
@@ -0,0 +1,94 @@
+import SwiftUI
+import Factory
+
+struct LoginView: View {
+
+    @State private var username = ""
+    @State private var password = ""
+
+    @FocusState private var emailIsFocused: Bool
+    @FocusState private var passwordIsFocused: Bool
+
+    var viewModel: LoginViewModel
+
+    var body: some View {
+        ZStack {
+            LinearGradient(gradient:
+                            Gradient(colors:
+                                        [Color(red: 0.453, green: 0.308, blue: 0.637),
+                                         Color(red: 0.154, green: 0.185, blue: 0.463)]),
```

Comment:
Can this be a private computed property?

Reply:
It is changed in the next PR (#32 )